desktop/apps/artsy_primer/routes.js

```diff
@@ -3,19 +3,22 @@ import _ from 'underscore'
 import moment from 'moment'
 import Article from '../../models/article'
 import { crop } from '../../components/resizer'
+import {
+  SAILTHRU_KEY,
+  SAILTHRU_SECRET,
+  SAILTHRU_MASTER_LIST
+} from '../../config.coffee'
 
-import { SAILTHRU_KEY, SAILTHRU_SECRET, SAILTHRU_MASTER_LIST } from '../../config.coffee'
-const sailThruClient = require('sailthru-client').createSailthruClient(SAILTHRU_KEY,SAILTHRU_SECRET)
+const sailThruClient = require('sailthru-client')
+  .createSailthruClient(SAILTHRU_KEY, SAILTHRU_SECRET)
 
 export const article = async (req, res, next) => {
-  if (!req.user) {
-    return res.redirect('/log_in?redirect_uri=' + req.url)
-  }
-
-  const article = new Article({id: req.params.id})
+  if (!req.user) return res.redirect('/log_in?redirect_uri=' + req.url)
+  const article = new Article({ id: req.params.id })
   await article.fetchWithRelated({
     accessToken: req.user && req.user.get('accessToken')
   })
+  if (req.url.match('primer-digest')) res.locals.h2 = 'Collecting Digest'
```

Comment:
This line is what gives us the new header copy—the rest is just noise from making the file pass Standard.